packages/next-swc/crates/core/tests/loader/css-hygiene-1/output.js

```diff
@@ -1,3 +1,3 @@
-var _defaultExport = new String("@media(max-width:870px){th.expiration-date-cell,td.expiration-date-cell{display:none}}");
+var _defaultExport = new String("@media (width<=870px){:is(:global(th.expiration-date-cell),:global(td.expiration-date-cell)){display:none}}");
```

Comment:
The `:global` in here looks suspicious

Reply:
Oops, thank you!

Reply:
Seems like it's a bug of `lightningcss`.

https://github.com/swc-project/plugins/pull/205#issuecomment-1700503233